src/appservice/config/config.production.yaml

```diff
@@ -0,0 +1,108 @@
+#
+# Note that this effectively makes the protectedRooms and associated commands useless
+# for regular rooms.
+#
+# Note: the management room is *excluded* from this condition.
+# Explicitly add it as a protected room to protect it.
+#
+# Note: Ban list rooms the bot is watching but didn't create will not be protected.
+# Explicitly add these rooms as a protected room list if you want them protected.
+protectAllJoinedRooms: false
+
+# Increase this delay to have Draupnir wait longer between two consecutive backgrounded
+# operations. The total duration of operations will be longer, but the homeserver won't
+# be affected as much. Conversely, decrease this delay to have Draupnir chain operations
+# faster. The total duration of operations will generally be shorter, but the performance
+# of the homeserver may be more impacted.
+backgroundDelayMS: 500
+
+# Misc options for command handling and commands
+commands:
+  # Whether or not the `!draupnir` prefix is necessary to submit commands.
+  #
+  # If `true`, will allow commands like `!ban`, `!help`, etc.
+  #
+  # Note: Draupnir can also be pinged by display name instead of having to use
+  # the !draupnir prefix. For example, "my_moderator_bot: ban @spammer:example.org"
+  # will address only my_moderator_bot.
+  allowNoPrefix: false
+
+  # Any additional bot prefixes that Draupnir will listen to. i.e. adding `mod` will allow `!mod help`.
+  additionalPrefixes:
+    - "draupnir"
+
+  # Whether or not commands with a wildcard (*) will require an additional `--force` argument
+  # in the command to be able to be submitted.
+  confirmWildcardBan: true
+
+  # The default reasons to be prompted with if the reason is missing from a ban command.
+  ban:
+    defaultReasons:
+      - "spam"
+      - "brigading"
+      - "harassment"
+      - "disagreement"
+
+# Configuration specific to certain toggle-able protections
+protections:
+  # Configuration for the wordlist plugin, which can ban users based if they say certain
+  # blocked words shortly after joining.
+  wordlist:
+    # A list of case-insensitive keywords that the WordList protection will watch for from new users.
+    #
+    # WordList will ban users who use these words when first joining a room, so take caution when selecting them.
+    #
+    # For advanced usage, regex can also be used, see the following links for more information;
+    #  - https://www.digitalocean.com/community/tutorials/an-introduction-to-regular-expressions
+    #  - https://regexr.com/
+    #  - https://regexone.com/
+    words:
+      - "LoReM"
+      - "IpSuM"
+      - "DoLoR"
+      - "aMeT"
+
+    # For how long (in minutes) the user is "new" to the WordList plugin.
+    #
+    # After this time, the user will no longer be banned for using a word in the above wordlist.
+    #
+    # Set to zero to disable the timeout and make users *always* appear "new".
+    # (users will always be banned if they say a bad word)
+    minutesBeforeTrusting: 20
```

Comment:
Hmm why did you duplicate this and not just tell them to reuse the one in `config/`?

Reply:
Its a subset of the one in config/ that only includes keys that are actually inherited. Well technically thats a lie verbose is not mentioned at all

Reply:
Hmm, i wish there was a way to do this without duplicating since the `default.yaml` kind of is the documentation, which tbh also sucks.

Reply:
Tbh i think that there is a better way to do this in the future but i would say lets leave that for later. Since i think this is a case where perfect is the enemy of good. Since if we want to be perfect the whole AS stuff should also be in the config dir.

Reply:
Perfect is the enemy of good is gonna go on the list of banned expressions. Since, that's the kind of dogma that has driven the ecosystem down the drain. 

Regardless, I'd rather delete the file and tell them which keys will be copied across.